content/en/04-tutorials/03-integrations/02-grafana.md

```diff
@@ -43,188 +43,262 @@ Before you can create a dashboard, you will need to add Pixie as a datasource:
 <svg title='' src='grafana/configure-plugin.png'/>
 :::
 
+<Alert variant="outlined" severity="info">
```

Comment:
The positioning of this looks a little weird. It looks closer to line 6 than 5. What if we move it above the image?